extension/src/content/index.js

```diff
@@ -46,6 +45,42 @@ function init() {
     yaml: "yaml"
   };
 
+  // Pasting events are used to force stackoverflows WMD to save state.
+  // TODO: This solution is buggy and does not always work - more discussion needed on solution
+  function stackOverflowFormat(
```

Comment:
This is only doing some formatting - the the bulk of the formatting is happening [here](https://github.com/prettier/prettier-chrome-extension/pull/49/files#diff-2b5769826b6670c8ab28f63985a301a8R276-R401).

Can we not abstract this out yet? Or have it be just the DOM mutation?